components/TextFields/src/MDCTextInputControllerTextFieldBox.m

```diff
@@ -112,6 +115,26 @@ - (UIEdgeInsets)textInsets:(UIEdgeInsets)defaultInsets {
   return textInsets;
 }
 
+- (void)updateLayout {
+  [super updateLayout];
+
+  if (!self.textInput) {
+    return;
+  }
+
+  if (!self.clearButtonBottom) {
+    self.clearButtonBottom = [NSLayoutConstraint constraintWithItem:self.textInput.clearButton
+                                                          attribute:NSLayoutAttributeBottom
+                                                          relatedBy:NSLayoutRelationEqual
+                                                             toItem:self.textInput.underline
+                                                          attribute:NSLayoutAttributeTop
+                                                         multiplier:1
+                                                           constant:-1 * ([self beneathInputPadding] - MDCTextInputClearButtonImageBuiltInPadding + MDCTextInputTextFieldBoxClearButtonPaddingAddition)];
```

Comment:
There is a chance someone would edit this value and wonder why it isn't affecting the rendering.
Consider setting constant to 1 with a comment that the constant value will be set below.